cmake/bazel_to_cmake.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+this_script_dir="$(dirname "$0")"
+
+root_dir="$(git -C "${this_script_dir}" rev-parse --show-toplevel)"
+
+build_files="$(find "${root_dir}" -type f -name BUILD)"
+
+if ! command -v python3 &> /dev/null
```

Comment:
Nit: probably good to follow Google bash style here: https://google.github.io/styleguide/shellguide.html#loops

Reply:
done